api/integrations/lead_tracking/hubspot/services.py

```diff
@@ -7,37 +7,37 @@
     HUBSPOT_ACTIVE_SUBSCRIPTION_SELF_HOSTED,
     HUBSPOT_COOKIE_NAME,
 )
-from users.models import HubspotTracker
+from users.models import FFAdminUser, HubspotTracker
 
 logger = logging.getLogger(__name__)
 
 
-def register_hubspot_tracker(request: Request) -> None:
+def register_hubspot_tracker(request: Request, user: FFAdminUser | None = None) -> None:
     hubspot_cookie = request.data.get(HUBSPOT_COOKIE_NAME)
-
+    track_user = user if user else request.user
     if not hubspot_cookie:
-        logger.info(f"Request did not included Hubspot data for user {request.user.id}")
+        logger.info(f"Request did not included Hubspot data for user {track_user.id}")
         return
 
     if (
         HubspotTracker.objects.filter(hubspot_cookie=hubspot_cookie)  # type: ignore[misc]
-        .exclude(user=request.user)
+        .exclude(user=track_user)
         .exists()
     ):
         logger.info(
-            f"HubspotTracker could not be created for user {request.user.id}"
+            f"HubspotTracker could not be created for user {track_user.id}"
             f" due to cookie conflict with cookie {hubspot_cookie}"
         )
         return
```

Comment:
@matthewelwell , I think we could remove this guard and allow for multiple identical cookies (cf the user who created 3 accounts. We lost the info on the account we cared about)